examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -1,28 +1,35 @@
 var fs = require('fs');
 var util = require('util');
 var request = require("request");
+var cmdline = require("./cmdline");
 
-var host = process.argv[2] || 'http-api-host';
-var port = 3000;
-var url = 'https://' + host + ':' + port;
-var NUM_REQUEST = 10;   // Number of long-poll requests before unsubscribe
+var NUM_REQUEST = 10; // Number of long-poll requests before unsubscribe.
 var counter = 0;
-var opt = {
-    cert: fs.readFileSync('client.crt'),
-    key: fs.readFileSync('client.key'),
-    ca: fs.readFileSync('bloomberg.crt')
-};
+
+function url()
```

Comment:
Ok then. I'll remove the functions (although I like functions) but I'll keep the explicit `parse` function in the `cmdline.js` module and I will call it before `opt` and `url` are defined.